apps/core/src/hooks/useTransactionSummary.ts

```diff
@@ -45,6 +50,140 @@ const getSummary = (
     }
 };
 
+const getObjectSummaryNFTData = <T extends { objectId: string }>({
+    nftsMeta,
+    originalData,
+}: {
+    nftsMeta: ReturnType<typeof useNFTsMeta>;
```

Comment:
Can we align on calling this `display` or something? This naming is very confusing